_posts/en/newsletters/2022-11-30-newsletter.md

```diff
@@ -0,0 +1,170 @@
+      through Bob's node, her node first uses LN to buy a credential
+      from Bob.
+
+    - *Previous success:* if a payment that Alice sent through Bob's
+      node is successfully accepted by the ultimate receiver, Bob's node
+      can return a credential token to Alice's node---or even more
+      tokens than were previously used, allowing Alice's node to send
+      additional value through Bob's node in the future.
+
+    - *UTXO ownership proofs or other alternatives:* although not
+      necessary for Riard's initial proposal, some forwarding nodes may
+      experiment with giving credentials to everyone who proves they own
+      a Bitcoin UTXO, perhaps with modifiers that give older or
+      higher-value UTXOs more credential tokens than newer or
+      lower-value UTXOs.  Any other criteria can be used as each
+      forwarding node chooses for itself how to distribute its
+      credential tokens.
+
+    Clara Shikhelman, whose own co-authored proposal partly based on
+    local reputation was described in [Newsletter #226][news226 jam],
+    replied to [ask][shikelman credentials] whether credential tokens
+    were transferable between users and whether that could lead to the
+    creation of a market for tokens.  She also asked how they would work
+    with [blinded paths][topic rv routing] where a spending node
+    wouldn't know the full path to the receiving node.
+
+    Riard [replied][riard double spend] that it would be difficulty to
+    redistribute credential tokens and create a market for them because
+    any transfer would require trust.  For example, if Bob's node
+    issues a new credential to Alice, who then tries to sell the
+    credential to Carol, there's no trustless way for Alice to prove she
+    won't try to use the token herself even after Carol has paid her.
+
+    For blinded paths, [it appears][harding paths] the receiver can
+    provide any necessary credentials in an encrypted form without
+    introducing a secondary vulnerability.
+
+    Additional feedback for the proposals was received on its related
+    [pull request][bolts #1043].
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Bitcoin Core 24.0][] is the next major version of the network's most
+  widely used full node implementation.  It contains several new
+  features and multiple bug fixes, as described in its [release
+  notes][bcc 24.0 rn].
+
+  **Warning:** this release includes the `mempoolfullrbf` configuration
+  option which several protocol and application developers believe could
+  lead to problems for merchant services as described in previous
+  newsletters [#222][news222 rbf] and [#223][news223 rbf].  It could
+  also cause problems for transaction relay as described in [Newsletter
+  #224][news224 rbf].
```

Comment:
It seems that 24.0 has some bugs and 24.0.1 is coming soon. Not sure if it's weird to remove this but AFAIK wouldn't really recommend people use it? cc @fanquake

Reply:
I would suggest delaying this until 24.0.1, which will ideally be out within a week.